app/models/incomeSourceDetails/viewmodels/ObligationsViewModel.scala

```diff
@@ -18,7 +18,7 @@ package models.incomeSourceDetails.viewmodels
 
 import java.time.LocalDate
 
-final case class ObligationsViewModel(quarterlyObligationsDatesYearOne: Seq[DatesModel], quarterlyObligationsDatesYearTwo: Seq[DatesModel],
+final case class ObligationsViewModel(quarterlyObligationsDates: Seq[Seq[DatesModel]],
```

Comment:
Should fix: I don't think there is any need for this Seq to be nested in another Seq

Reply:
With this I thought the outer Seq would hold the relevant years while the inner Seq holds the DateModel instances within that year?

Reply:
It looks like it'd just be Seq(Seq(DatesModel, DatesModel)), I think it could be achieved with just 1 Seq